### PR TITLE
Fix testadjustcf.R documentation

### LINKED ISSUE #8: Update testadjustcarryforward.R documentation and output
A couple of things that would improve ease of use of the testadjustcarryforward.R:

- Update the documentation to specify the difference in the command line argument between Macs and Windows (no `exec/` in Macs)
- Update output to appear in working directory by default, with a timestamp in the name. Users can otherwise specify the file name.

### PR DESCRIPTION
**Pull request by delosh653**
Fixes #8 (Update testadjustcarryforward.R documentation and output).

### CODE REVIEW

File: README-adjustcarryforward.md [hunk 1]
```diff
@@ -31,25 +31,32 @@ file:
...
+Windows as follows:
 
 ```bash
 % Rscript exec/testadjustcf.R cleaned.csv
 ```
 
+For Macs/Linux, the script is executed with the following command:
+
+```bash
+% Rscript testadjustcf.R cleaned.csv
```
Comment (dchud) [inline README-adjustcarryforward.md]: When I run this script on my Mac, I use the full relative path, including `exec/`. I'm trying to recall, was there some adjustment David made to his version other than changing directories? 

Either way, do we need to say something about the current working directory?

File: exec/testadjustcf.R [hunk 2]
```diff
@@ -189,8 +190,8 @@ if (args$maxrecs > 0) {
   dm_filt <- dm
 }
 
-# Create outdir if necessary; ignore warnings if it already exists
-dir.create(file.path(outdir), showWarnings = FALSE)
+# # Create outdir if necessary; ignore warnings if it already exists
```
Comment (dchud) [inline exec/testadjustcf.R]: Should these two commented lines be removed?

File: README-adjustcarryforward.md (hunk 1 above)
Comment (delosh653) [reply README-adjustcarryforward.md]: Hmmm, now I'm wondering that as well. He might have just navigated to that directory then? I will update this and say something about the working directory, I agree that it's important.

File: exec/testadjustcf.R (hunk 2 above)
Comment (delosh653) [reply exec/testadjustcf.R]: Yes! Will do, thanks for catching that!

**Comment by dchud** [comment]
Rebased here, created new PR at https://github.com/carriedaymont/growthcleanr/pull/15 to merge over there.

**Pull request closed without merging**